docs/sphinx/manuals/roc_recv.rst

```diff
@@ -122,17 +118,23 @@ Listen on two ports on a particular interface:
 
     $ roc-recv -vv -s rtp+rs8m:192.168.0.3:10001 -r rs8m:192.168.0.3:10002
 
-Listen on two Reed-Solomon ports, two LDPC ports, and one bare RTP port:
+Listen on two ports on a particular multicast interface (but not join to a multicast group):
+
+.. code::
+
+    $ roc-recv -vv -s rtp+rs8m:225.1.2.3:10001 -r rs8m:225.1.2.3:10002
+
+Listen on two ports on a particular multicast interface and join to a multicast group on all interfaces:
```

Comment:
AFAIK there is no such thing as "multicast interface".

`multicast interface` -> `multicast address`

Reply:
I wanted to keep things the same. We have:

```rst
Listen on two ports on a particular interface:

.. code::

    $ roc-recv -vv -s rtp+rs8m:192.168.0.3:10001 -r rs8m:192.168.0.3:10002
```

I considered that having an example in which we also *Listen on two ports on a particular interface* but this interface corresponds to the multicast address.